package.json

```diff
@@ -392,6 +392,21 @@
                     "default": false,
                     "description": "Set to true to show timestamps by default in the log viewer."
                 },
+                "vscode-kubernetes.log-viewer.since": {
+                    "type": "integer",
```

Comment:
I thought the `since` field took a string so it could have units, e.g. `5m` or `2h`?  If not, we need to say what the units are for the config setting.

Reply:
Please check this:
https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/pull/1074#discussion_r874228172

Yeah I wasn't 100% sure how to handle this part tbh.
I didn't specify units since technically, it's accompanied by another field which is the units, so it's a bit confusing. 🤔 

I'll think about this.

![image](https://user-images.githubusercontent.com/22801583/168697106-eabbfd25-ae28-49c8-a865-89bbaf42d8cf.png)